README.md

```diff
@@ -4,3 +4,26 @@ Installers for MNE-Python for macOS, Windows, and Linux.
 
 ⚠️ Nothing to see here for now, please move along and check back again in a few
 days. ⚠️
+
+
+
+## Installation
+
+To install MNE-Python simply double click the installer.
+You can customise some parameters, but the default values should suit most users.
```

Comment:
American spelling please :) I know some folks living outside the USA dislike this, but I'm aiming for consistency, and in computer-related stuff, American seems to be dominant 😅